src/test/java/gregtech/overclock/GT_OverclockCalculator_UnitTest.java

```diff
@@ -78,7 +78,10 @@ void perfectAndImperfectOverclockEBFWithTwoHeatDiscounts_Test() {
             .setRecipeHeat(1800)
             .setMachineHeat(3600)
             .calculate();
-        assertEquals(1024 / (Math.pow(4, 1) * Math.pow(2, 3)), calculator.getDuration(), messageDuration);
+        assertEquals(
+            1024 / (GTUtility.powInt(4, 1) * GTUtility.powInt(2, 3)),
+            calculator.getDuration(),
+            messageDuration);
```

Comment:
Not you, but calling pow for a power of 1 seems wild
```suggestion
        assertEquals(
            1024 / (4.0D * GTUtility.powInt(2, 3)),
            calculator.getDuration(),
            messageDuration);
```